[FEATURE] Holding shift while using map zoom tools inverts the zoom

nyalldawson opened the pull request:
Eg hold shift and click with zoom in tool = zoom out

I used shift to match composer behaviour. But maybe alt is a better choice for both. Shift feels a bit odd to me...

@nirvn? @DelazJ ?

nyalldawson committed:
[FEATURE] Holding shift while using map zoom tools inverts the zoom

Eg hold shift and click with zoom in tool = zoom out

nirvn commented:
@nyalldawson , agreed, alt would be better fitted here. The composer shortcut should be updated too if possible.

nyalldawson committed:
[needs-docs] Use alt modifier to reverse zoom behaviour

Previously shift-modifier was used to change zoom in to out, and
zoom out to in. This commit changes it to the more common alt
modifier.

nyalldawson commented:
Done, changed both to alt

dakcarto commented:
@nyalldawson, thanks for this! On Mac, this is generally the Option (Alt) key as well.

nyalldawson commented:
Thanks @dakcarto for the confirmation - I've merged.
